tests/components/proximity/test_init.py

```diff
@@ -813,19 +582,27 @@ async def test_device_tracker_test1_nearest_after_test2_in_ignored_zone(
     )
     await hass.async_block_till_done()
 
-    assert await async_setup_component(
+    await async_setup_single_entry(
         hass,
-        DOMAIN,
-        {
-            "proximity": {
-                "home": {
-                    "ignored_zones": ["work"],
-                    "devices": ["device_tracker.test1", "device_tracker.test2"],
-                    "zone": "home",
-                }
-            }
-        },
-    )
+        "zone.home",
+        ["device_tracker.test1", "device_tracker.test2"],
+        ["zone.work"],
+        1,
+    )
+
+    # assert await async_setup_component(
```

Comment:
Commented code.

Reply:
nice catch ... PR is on the way, thx :+1:

Reply:
- #123500